package.json

```diff
@@ -43,5 +43,9 @@
         "sinon": "^13.0.1",
         "webpack": "^5.73.0",
         "webpack-cli": "^4.9.2"
-    }
+    },
+    "browserslist": [
+        "defaults",
+        "IE 8"
```

Comment:
**Question:** why go with `IE 8` here whereas in the `webpack.config.js` file you targeted `IE 10`?
https://github.com/mozmeao/cookie-helper/pull/7/files#diff-1fb26bc12ac780c7ad7325730ed09fc4c2c3d757c276c3dacc44bfe20faf166fR30

Reply:
Ah good catch! This was copied over from bedrock, where we have different targets for webpack and babel. We can simplify here, will update.